pg_backup_api/news.md

```diff
@@ -2,6 +2,15 @@
 
 © Copyright EnterpriseDB UK Limited 2021-2023 - All rights reserved.
 
+## 1.2.1 (2023-07-10)
```

Comment:
I think because this is introducing more than just bug fixes the version should be `1.3.0` rather than `1.2.1`.